samples/SkijaInjectSample/src/main/kotlin/SkijaInjectSample/App.kt

```diff
@@ -1,35 +1,26 @@
 package SkijaInjectSample
 
-import org.jetbrains.skiko.SkiaWindow
-
-import java.awt.event.MouseEvent
-import javax.swing.WindowConstants
-import javax.swing.event.MouseInputAdapter
 import org.jetbrains.skija.*
-import org.jetbrains.skiko.SkiaRenderer
-import java.awt.event.MouseMotionAdapter
-import kotlin.math.cos
-import kotlin.math.sin
 import org.jetbrains.skija.paragraph.FontCollection
 import org.jetbrains.skija.paragraph.ParagraphBuilder
 import org.jetbrains.skija.paragraph.ParagraphStyle
 import org.jetbrains.skija.paragraph.TextStyle
-import java.awt.event.ActionEvent
-import java.awt.event.ActionListener
-import java.awt.event.KeyEvent
+import org.jetbrains.skiko.SkiaLayer
+import org.jetbrains.skiko.SkiaRenderer
+import org.jetbrains.skiko.SkiaWindow
 import java.awt.Toolkit
-import javax.swing.JFrame
-import javax.swing.JMenu
-import javax.swing.JMenuBar
-import javax.swing.JMenuItem
-import javax.swing.JOptionPane
-import javax.swing.KeyStroke
+import java.awt.event.*
+import javax.swing.*
+import kotlin.math.cos
+import kotlin.math.sin
 
 fun main(args: Array<String>) {
-    createWindow("First window")
+    repeat(1) {
+        createWindow("window $it")
+    }
 }
 
-fun createWindow(title: String) {
+fun createWindow(title: String) = SwingUtilities.invokeLater {
```

Comment:
In `SwingSkia.kt` you also need to change `fun SwingSkia() = SwingUtilities.invokeLater`

Reply:
Fixed